src/Homotopy/Truncation/SetTruncation.ard

```diff
@@ -0,0 +1,14 @@
+\import Homotopy.Truncation
+
+\data SetTrunc (A : \Type)
+  | point A
+  | trunc (x y : SetTrunc A) (p q : x = y) (i j : I) \elim i, j {
+    | left, j => p @ j
+    | right, j => q @ j
+    | i, left => x
+    | i, right => y
+  }
+
+\instance SetTruncTruncated {A : \Type} : Truncated_-1+ 1 (SetTrunc A)
+  | isTruncated x y p q => path (\lam i => path (trunc x y p q i))
```

Comment:
I wish I can prove this via:

```arend
path (\lam i => path (path (trunc x y p q) @ i))
```

Or:

```arend
pmap path (path (trunc x y p q))
```

Or:

```arend
pmap (\lam i => path i) (path (trunc x y p q))
```

But non of these works, they fail with some type inferences where only one candidates available